app/models/local_restriction.rb

```diff
@@ -0,0 +1,43 @@
+class LocalRestriction
+  FILE_PATH = "app/lib/local_restrictions/local-restrictions.yaml".freeze
+
+  attr_reader :gss_code
+
+  def initialize(gss_code)
+    @gss_code = gss_code
+  end
+
+  def area_name
+    restriction["name"]
+  end
+
+  def alert_level
+    restriction["alert_level"]
+  end
+
+  def guidance
+    restriction["guidance"]
+  end
+
+  def extra_restrictions
+    restriction["extra_restrictions"]
+  end
+
+  def start_date
+    restriction["start_date"].to_date
```

Comment:
I think we need to add tests for when start_date and end_date is nil:

```
irb(main):005:0> rst = LocalRestriction.new("E06000008")
=> #<LocalRestriction:0x0000562becd45eb8 @gss_code="E06000008">
irb(main):006:0> rst.start_date
Traceback (most recent call last):
        2: from (irb):6
        1: from app/models/local_restriction.rb:27:in `start_date'
NoMethodError (undefined method `to_date' for nil:NilClass)
Did you mean?  to_d
```

It should be possible to fix this with a safe navigation operator, e.g.:

```
restriction["start_date"]&.to_date
```

Reply:
ooh good spot, sorry! Been fixed now and tests added :)